src/sections/Navbar.html

```diff
@@ -0,0 +1,26 @@
+<nav
+  class="flex flex-row justify-between items-center md:px-8 p-4 lg:p-6 xl:p-6 font-outfit"
+>
+  <div class="flex flex-col md:text-2xl lg:text-2xl">
+    <p>DE</p>
+    <p>EN</p>
+  </div>
+  <div class="lg:absolute lg:pl-[40%]">
+    <h1 class="font-semibold text-2xl md:text-5xl">Baumeister</h1>
+  </div>
+  <div
+    class="absolute lg:flex flex-col items-end hidden md:ml-[20%] lg:ml-0 pt-56 lg:pt-14 pl-32 lg:pl-[88%] font-outfit text-2xl text-green-700 lg:text-black"
+    id="menu"
+  >
+    <p>About</p>
+    <p>Services</p>
+    <p>References</p>
+    <p>Contact</p>
+  </div>
```

Comment:
The text for the navbar elements should be bigger and bolder and the gap should be bigger too.